gatling-s3-reporter/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM openjdk:8u212-jdk-alpine
+
+ARG AWS_VERSION=1.16.58
+ARG GATLING_VERSION=2.2.3
```

Comment:
@kamegu 
https://github.com/chatwork/dockerfiles/blob/master/helm/variant.mod
https://github.com/chatwork/dockerfiles/blob/master/helm/variant.lock
If you need to update to the latest version automatically, add `variant.mod` and `variant.lock`.